>From 11b46ca41989e3fb9e089dcb1d042046d406c05a Mon Sep 17 00:00:00 2001
From: Antti-Juhani Kaijanaho <ajk@debian.org>
Date: Thu, 28 Jun 2012 21:08:29 +0300
Subject: [PATCH] tests/0022.*: New failing test case for join-dctrl

The test exposes a missing feature (missing -o).

Signed-off-by: Antti-Juhani Kaijanaho <ajk@debian.org>
---
 tests/0022.in  |    2 ++
 tests/0022.out |    2 ++
 tests/0022.sh  |    8 ++++++++
 3 files changed, 12 insertions(+)
 create mode 100644 tests/0022.in
 create mode 100644 tests/0022.out
 create mode 100644 tests/0022.sh

diff --git a/tests/0022.in b/tests/0022.in
new file mode 100644
index 0000000..67a9cc1
--- /dev/null
+++ b/tests/0022.in
@@ -0,0 +1,2 @@
+Mergefield: 1
+Duplicate-field: foo
diff --git a/tests/0022.out b/tests/0022.out
new file mode 100644
index 0000000..67a9cc1
--- /dev/null
+++ b/tests/0022.out
@@ -0,0 +1,2 @@
+Mergefield: 1
+Duplicate-field: foo
diff --git a/tests/0022.sh b/tests/0022.sh
new file mode 100644
index 0000000..097ecd0
--- /dev/null
+++ b/tests/0022.sh
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+set -e
+
+LC_ALL=C
+export LC_ALL
+
+$JOIN_DCTRL -j Mergefield 0022.in 0022.in
-- 
1.7.10

